indigo_app/templates/indigo_api/_work_filter_form.html

```diff
@@ -0,0 +1,57 @@
+{% load account %}
+
+
+<form method="GET" id="work-filter-form">
+  <div class="d-flex">
+
+    <div class="flex-grow-1 mr-2">
+        <input type="text" class="form-control filter-search" name="search_string" placeholder="Filter by title, year or number" value="">
+    </div>
+
+    <!-- TODO: Types filter -->
+    <div class="mr-2">
+        <select class="form-control filter-subtype" name="subtype">
+          <option value="all">All types</option>
```

Comment:
These need to be dynamic and remember what has been selected, so best is to iterate over them:

```
{% for option in form.subtype %}
  {{ option }}
{% endfor %}
```

Reply:
Similarly for the option select fields.